README.md

```diff
@@ -44,8 +44,18 @@ and all subdirectories, and then check if there are cycles.
 If there are any, it will exit non-zero and print the files that are part of the cycle.
 Otherwise, it will not print anything and exit with zero.
 
-However, this is (currently) not configurable in any way,
-hence it is recommended to use the programmatic approach as described below.
+If you wish to overwrite the default settings you can pass the following parameters:
+
+```bash
+--checkDirs # list of relative or absolute paths
+--root # absolute path
```

Comment:
I think these are all relative-only, no? I don't think we ever want to hand over absolute paths, as this makes the setup of the arguments hard-coded to the consumers system